### PR TITLE
TOOLS-2557 Update eng submodule for new Rust Makefiles.

### PR DESCRIPTION
**Pull request by travispaul**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[workspace]
+members = [
+    "mako_rollup"
+]
```
Comment (travispaul) [inline Cargo.toml:4]: This is necessary for the Rust Makefile's `check` target to find `mako_rollup`.

File: mako_rollup/src/main.rs [hunk 2]
```diff
@@ -72,7 +73,9 @@ fn main() {
         }
     }
 
-    println!("# HELP used_bytes The current number of bytes used on a mako\n# TYPE used_bytes gauge");
+    println!(
+        "# HELP used_bytes The current number of bytes used on a mako\n# TYPE used_bytes gauge"
+    );
```
Comment (travispaul) [inline mako_rollup/src/main.rs:78]: `rust fmt`

File: Cargo.toml (hunk 1 above)
Comment (danmcd) [reply Cargo.toml:4]: You should make sure someone with more Rust clue signs off on this, then.

**Review by danmcd — COMMENTED** [review]
Looks like PR build is still failing similarly to `master`.

**Review by danmcd — APPROVED** [review]
I still approve, esp that both PR *and BEFORE THIS CHANGE* builds succeeded.  Thank you!

**Pull request merged**